petclinic-frontend/src/features/inventories/InventoryProducts.tsx

```diff
@@ -89,21 +89,37 @@ const InventoryProducts: React.FC = () => {
   ): Promise<void> => {
     if (currentQuantity > 0) {
       try {
-        // Send a PATCH/PUT request to update the quantity in the backend
+        // Calculate the updated quantity
         const updatedQuantity = currentQuantity - 1;
+
+        // Send the PATCH request to update the quantity in the backend
         await axios.patch(
-          `http://localhost:8080/api/gateway/inventory/${inventoryId}/products/${productId}`,
+          `http://localhost:8080/api/gateway/inventory/${inventoryId}/products/${productId}/consume`,
```

Comment:
Oh wow, the issue was you forgot to put /consume to your api request. Good job spotting this!